generators/cleanup.js

```diff
@@ -207,4 +207,7 @@ function cleanupOldServerFiles(generator, javaDir, testDir, mainResourceDir, tes
         generator.config.delete('blueprint');
         generator.config.delete('blueprintVersion');
     }
+    if (generator.isJhipsterVersionLessThan('6.4.0')) {
```

Comment:
As this is not server file then this should go to function `cleanupOldFiles` instead of `cleanupOldServerFiles`.

Reply:
Sorry about that.
Didn't noticed.